working/wildcards/feature-specification.md

```diff
@@ -99,8 +99,8 @@ A *local declaration* is any of:
 *   For loop variable declarations.
 
     ```dart
-    for (_ = 0;;) {}
-    for (_ in list) {}
+    for (int _ = 0;;) {}
+    for (var _ in list) {}
```

Comment:
I changed this because we want to see the declaration named `_`. With the previous version the name `_` could resolve to a top-level variable and several other things.